File: 2- First Year (2021-2022)/Term 1/Intro To Computing (CS211)/Lab/Week 10 - C-Strings/Task 2/Task 2.cpp
#include <iostream>
#include <string>
using namespace std;

int main()
{
	string first, middle, last, age, password = "";
	cout << "First Name: ";
	getline(cin, first);
	cout << "Middle Name: ";
	getline(cin, middle);
	cout << "Last Name: ";
	getline(cin, last);
	cout << "Age: ";
	getline(cin, age);
	
	password += first.at(first.length()/2); //add to password, dont reassign to it.
	password += middle.at(middle.length()/2);
	password = password + last;
	password = password + age;
	cout << "Password: " << password;
	
	return 0;
}

File: 3- Second Year (2022-2023)/Term 1/OOP1 (CS311)/Lab/Labs/Lab 3/Task2/src/Task2.java
/*Design and implement a Java program that generates a password for a student using 
his name and age as the following:
– Take the first letter from the first name, the middle letter from the 
middle name and the last letter from the last name.
– Then concatenate the obtained letters along with age which should be 
multiplied by 100.
– All letters should be in uppercase 
 */

import java.util.Scanner;
public class Task2 {
    public static void main(String[] args) throws Exception {
        String password = "";
        Scanner input = new Scanner(System.in);
        System.out.println("First Name: ");
        String firstName = input.next();
        System.out.println("Middle Name: ");
        String middleName = input.next();
        System.out.println("Last Name: ");
        String lastName = input.next();
        System.out.println("Age: ");
        int age = input.nextInt();
        password += firstName.charAt(0);
        password += middleName.charAt(middleName.length()/2);
        password += lastName.charAt(2);
        password += age*100;
        System.out.println("Your Password: " + password.toUpperCase());
        input.close();
    }
}
